dependencies/action.yml

```diff
@@ -0,0 +1,114 @@
+name: "Install ArmoniK dependencies"
+description: "Action to install dependencies required to deploy or use ArmoniK"
+inputs:
+  aws:
+    description: "Install dependencies for AWS"
+    type: boolean
+    required: false
+    default: false
+  docker:
+    description: "Install docker"
+    type: boolean
+    required: false
+    default: false
+  k3s:
+    description: "Install K3s"
+    type: boolean
+    required: false
+    default: false
+  terraform:
+    description: "Install Terraform"
+    type: boolean
+    required: false
+    default: false
+  AWS_ACCESS_KEY_ID:

```

Comment:
I think there is also a type or option for secrets.

Reply:
There is for nested workflows, but not for custom actions.

Reply:
desapointing...